### ISSUE TITLE
Anaconda licensing change

### ISSUE THREAD

**Issue by david-lippert** [gw-ospo/python-dev-tools-workshop-2024#4]
Not sure what our approach is for teaching Anaconda if it is no longer freely available.  https://www.theregister.com/2024/08/08/anaconda_puts_the_squeeze_on/

**Comment by david-lippert** [comment]
From Zach Chandler of the Stanford OSPO:

OK, so there are 3 contravening themes that argue against paying up:
1. This message from [anaconda.org](http://anaconda.org/) cofounder, clarifying the news: https://www.linkedin.com/feed/update/urn:li:activity:7229549723462905856/
2. Fully open, community based alternatives, like https://conda-forge.org/ which is not the same thing, have distros that many teams need via miniforge
3. Some of our RSEs say the GUI is a crutch and prevents people from learning important skills, that important features are only available via CLI, and they advocate for steering people away from [acaconda.org](http://acaconda.org/) (vocal minority)

**Comment by s2t2** [comment]
I think this only applies to anaconda.org distributions?

> Anaconda.org allows anyone to distribute their conda and standard Python packages to the world.

Not too familiar with those. I only use anaconda for virtual environments.

**Comment by labarba** [comment]
Please see the reply from CEO Peter Wang
https://www.linkedin.com/posts/pzwang_hi-everyone-recently-there-has-been-discussion-activity-7229549723462905856-rQH-

**Comment by labarba** [comment]
TL;DR "Anaconda's installers & package repos are free for teaching, learning, and research at accredited educational institutions worldwide."

**Comment by labarba** [comment]
> Some of our RSEs say the GUI is a crutch and prevents people from learning important skills, that important features are only available via CLI…

This is BS.
